src/coll_score/ucc_coll_score_map.c

```diff
@@ -66,9 +78,12 @@ ucc_status_t ucc_coll_score_map_lookup(ucc_score_map_t      *map,
                                        ucc_base_coll_args_t *bargs,
                                        ucc_msg_range_t     **range)
 {
-    ucc_memory_type_t mt      = ucc_coll_args_mem_type(bargs);
+    ucc_memory_type_t mt      = ucc_coll_args_mem_type(&bargs->args,
```

Comment:
minor, maybe change input parameter from ucc_bace_coll_args to ucc_coll_args since base_args is not needed here

Reply:
it's same pointer in the end of the day. i wanted to have change smaller. maybe we will need to pass smth else from base_args. i would keep it.